src/index.js

```diff
@@ -1,12 +1,45 @@
 // eslint-disable-next-line
 import _ from 'lodash';
 import './style.css';
+// import Storage from './storage.js';
 
-function component() {
-  const element = document.createElement('div');
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-  return element;
-}
-document.body.appendChild(component());
\ No newline at end of file
+// document.addEventListener('DOMContentLoaded', () => Storage.displaytodo());
+// document.addEventListener('DOMContentLoaded', () => Storage.checkboxevent());
+
+const mylocaldata = [{ indexed: 1, discribtion: 'Task-1', complete: false }, { indexed: 1, discribtion: 'Task-1', complete: false },
+  { indexed: 1, discribtion: 'Task-1', complete: false }];
+
+const todo = document.querySelector('.todo-list');
+mylocaldata.forEach((tododata, i) => {
+  const div = document.createElement('div');
+  div.id = `div${i}`;
+
+  const span1 = document.createElement('span');
+  span1.className = 'desc';
+  div.appendChild(span1);
+
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  span1.appendChild(checkbox);
+
+  const span2 = document.createElement('span');
+  span2.className = 'show';
+  span2.innerHTML = mylocaldata[i].discribtion;
+  span1.appendChild(span2);
+
+  const span3 = document.createElement('span');
+  span3.className = 'more';
+
+  const btn2 = document.createElement('span');
+  btn2.innerHTML = `
+  <i class="bi-three-dots-vertical" />        
+  `;
+  span3.appendChild(btn2);
+  div.appendChild(span3);
+  todo.appendChild(div);
+});
```

Comment:
- [ ] Ouch! this is close but as specified in the project requirements the list should appear in order of the index values for each task. So when you iterate over each task you have to make sure they are sorted by their index values first.
- Project requirement 👇
  ![image](https://github.com/coker2019/To-do-List/assets/104968236/1edac2f5-9343-40a7-ae16-3b176eacd02e)
- I noticed you've placed the same index values for each task please make sure the index value for each task object is different as they'll be used to order the items.

Reply:
## Okay, really sorry, I will fix it